lib/matplotlib/pyplot.py

```diff
@@ -1566,6 +1566,130 @@ def subplots(
     return fig, axs
 
 
+def subfigure_mosaic(
+    mosaic: str | HashableList,
+    *,
+    width_ratios: ArrayLike | None = None,
+    height_ratios: ArrayLike | None = None,
+    empty_sentinel: Any = '.',
+    subfigure_kw: dict[str, Any] | None = None,
+    gridspec_kw: dict[str, Any] | None = None,
+    per_subfigure_kw: dict[Hashable, dict[str, Any]] | None = None,
+    **fig_kw
+) -> tuple[Figure, dict[Hashable, matplotlib.axes.Axes]]:
+    """
+    Build a layout of SubFigures based on ASCII art or nested lists.
+
+    This is a helper function to build complex GridSpec layouts visually.
+
+    See :doc:`/gallery/subplots_axes_and_figures/subfig_mosaic`
```

Comment:
This does not appear to have been added?

Reply:
Yes, I haven't added the example yet. Working on it, though!